Monika After Story/game/0utils.rpy

```diff
@@ -647,6 +703,56 @@ python early in mas_utils:
         return False
 
 
+    def _get_version(ver_str, nums_only, split_nums=False):
+        """
+        Gets the version from a ver str
+
+        IN:
+            ver_str - version string to get version from
+            nums_only - True to only get ver numbers, False to get full ver
+
+        RETURNS: version str
+        """
+        if nums_only:
+            ver_nums = ver_str.partition("-")[0]
+        
+            if split_nums:
+                return ver_nums.split(".")
+
+            return ver_nums
+
+        return ver_str
+
+
+    def is_ver_stable(ver_str):
+        """
+        Checks if a version number is stable or not.
+        A stable version is generally a 3-tiered version number.
+
+        IN:
+            ver_str - version number string to check
+
+        RETURNS: true if version is stable, False if not.
+        """
+        return len(_get_version(ver_str, True, True)) == 3
+
+
+    def _is_downgrade(from_ver_str, to_ver_str):
+        """
+        Checks if the version transition given is a downgrade
+
+        IN:
+            from_ver_str - starting version (as ver str)
+            to_ver_str - ending version (as ver str)
+
+        RETURNS: true if downgrade, False if not
+        """
+        return compareVersionLists(
+            _get_version(from_ver_str, True, True),
+            _get_version(to_ver_str, True, True)
+        ) > 0
```

Comment:
I don't like this and think it's incorrect. Yes, we get valid output from this function, but it's really unclear and may change in future python versions. The problem is `_get_version` returns a `list` of **`str`**. But `compareVersionLists` uses comparison operators `>` and `<`. We should convert output of `_get_version` into `int` before passing them into `compareVersionLists`.

Can do it like `map(int, ver_list)`.